src/pages/Collection/index.tsx

```
@@ -508,68 +1361,106 @@ export const Collection: React.FC = () => {
+                </Stack>
+              </Grid>
             </Grid>
-          </Grid>
+          </div>
+        ) : (
+          <Container maxWidth="lg">
+            <Stack sx={{ mt: 5 }} gap={2}>
+              <Skeleton variant="text" width={280} height={50} />
+              <Grid container spacing={2}>
+                {[1, 2, 3, 4, 5, 6].map((el) => (
+                  <Grid item xs={6} sm={4} md={3} lg={2}>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)